src/parser/mod.rs

```diff
@@ -1141,13 +1141,36 @@ impl<'a, 'o> Parser<'a, 'o> {
             &self.options.extension.front_matter_delimiter,
         ) {
             if let Some((front_matter, rest)) = split_off_front_matter(s, delimiter) {
+                let lines = front_matter
+                    .as_bytes()
+                    .iter()
+                    .filter(|b| **b == b'\n')
+                    .count();
+
+                let mut stripped_front_matter = front_matter.to_string();
+                strings::remove_trailing_blank_lines(&mut stripped_front_matter);
```

Comment:
I'm concerned by this line. I'm experiencing an off-by-one error and there is space between my frontmatter and the beginning of my work. This would account for it.

Reply:
@SamWilsn